registry/storage/manifeststore.go

```diff
@@ -1,163 +0,0 @@
-package storage
-
-import (
-	"fmt"
-
-	"github.com/docker/distribution"
-	"github.com/docker/distribution/context"
-	"github.com/docker/distribution/digest"
-	"github.com/docker/distribution/manifest/schema1"
-	"github.com/docker/distribution/reference"
-	"github.com/docker/libtrust"
-)
-
-type manifestStore struct {
```

Comment:
I'm not sure if I'm on board with this file rename. The struct doesn't seem to be renamed. Do we really only store signed manifests here?